### PR TITLE
Add a list of banned words for submissions

### PR DESCRIPTION
**Pull request by jkampich1411**
Stole the list of banned words from: http://www.bannedwordlist.com/
If I should add more "bad words" reply here!

### CODE REVIEW

**Comment by jkampich1411** [comment]
would fix:
#30 (partially #23 if more moderation is added!)

File: backend/interfaces/ideas/router.py [hunk 1]
```diff
@@ -62,6 +63,10 @@ async def get_idea(id: int, db: Session = Depends(get_db)):
 
 @router.post('', response_model=IdeaResponse)
 async def create_idea(idea_request: CreateIdea, db: Session = Depends(get_db)):
+    for word in BANNEDWORDS:
+        if idea_request.title == word or idea_request.description == word or idea_request.tags == word:
```
Comment (bykof) [inline backend/interfaces/ideas/router.py]: i would rather check it like this:

```
for word in BANNEDWORDS:
    lower_word = word.lower()
    if lower_word in idea_request.title.lower() or lower_word in idea_request.description.lower() or lower_word in idea_request.tags.lower():

```

Comment (jkampich1411) [reply backend/interfaces/ideas/router.py]: oh yeah that makes more sense,
let me fix that quickly!

Comment (jkampich1411) [reply backend/interfaces/ideas/router.py]: fixed!

**Pull request merged**